releases/WD-2016-07-14.html

```diff
@@ -0,0 +1,657 @@
+<!doctype html><html lang="en">
```

Comment:
I'm not sure why this file is needed at all? These documents are all on the W3C server. Additionally, seems redundant to include them in a release/ folder. If anything, just `git tag` the appropriate release with a date. It's much more useful than having redundant generated files in a release folder.